### DISCUSSION TITLE
Welcome to simkube Discussions!

### DISCUSSION THREAD

**Discussion by drmorr0** [acrlabs/simkube#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by echarlesgrasby** [comment]
Hi there - I'm Eric- a developer from the great state of Arkansas! I saw your presentation during KubeCon 2023 and I was very impressed with the product that you have built. I think that your idea of having a simulator for running counterfactual scenarios in Kubernetes traffic is intriguing. 

Professionally, I am going to talk to my management at my current company and see where we could fit simkube in our Kubernetes deployments. We are on the outset of standing up our k8s infrastructure- and this would be an ideal time to consider adding this to our workflow. I'll reach out to you via email if I can get any traction with that.

Personally, if I can be of any help as a volunteer- sign me up. I'd be happy to help work on any features (I've wanted to learn Rust anyway, and have a bit of golang experience), or even help build out end user documentation. Anything to help a good open source project and fellow academian :)

**Reply by drmorr0** [reply]
Hi!  Thanks so much for the kind words.  Would love to chat more if you're able to incorporate simkube into your work.

In terms of contributions, I'm happy to have any contributions.  Probably the place to start would be trying to get the project set up in your local dev environment.  If you find any issues with the docs, would welcome PRs.

I don't know what to do for issue tracking yet.  So I have a list of issues written down internally but I really should persist those somewhere publicly available.  Lemme figure out how to do that and I'll follow up.

**Reply by drmorr0** [reply]
@echarlesgrasby OK -- I have dumped a whole bunch of "known issues" into the Github issues for this project, if there's something in there that piques your interest I would love some help.  I also have a bunch of "future work/feature requests/proposals" that I stuck in the Github Discussions, if there's anything in there that is of interest and you want to either comment on the discussion or start moving towards an actual implementation of something, please feel free as well.
